keyboards/yetis/keymaps/default/rules.mk

```diff
@@ -0,0 +1,3 @@
+MOUSEKEY_ENABLE = yes
+
+SRC += users/callum/swapper.c users/callum/oneshot.c
```

Comment:
Userspace are no longer included in the upstream QMK repository for various maintenance reasons, so this line must be removed. 
```suggestion
```

Reply:
done